### ISSUE TITLE
Unknown entity 'all'

### ISSUE THREAD

**Issue by tjorim** [frenck/spook#16]
Seems like Spook is being a bit strict on this:

```
Unknown entities used in: Turn off lights and heating when nobody is home
Spook has found a ghost in your automations 👻

While floating around, Spook crossed path with the following automation:

Turn off lights and heating when nobody is home (automation.turn_off_lights_when_nobody_is_home)

This automation references the following entities, which are unknown to Home Assistant:

all
To fix this error, edit the automation and remove the use of these non-existing entities.

Spook 👻 Not your homie.

Warning - 2/28/2023
```

Afaik `all` is a valid 'entity' (and the automation seems to works just fine).

Spook v0.10.100
Home Assistant 2023.3.0b5
Supervisor 2023.01.1
Operating System 9.5
Frontend 20230224.0 - latest

**Comment by frenck** [comment]
Nice! That is a small and simple bug. Thanks for reporting :)

**Comment by frenck** [comment]
Fixed in [5f2f779](https://github.com/frenck/spook/commit/5f2f77948a37df58289687b06ad2c08fbb4233dc)
